namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -430,7 +429,7 @@ public boolean hasAccess(Group group, UUID player, PermissionType perm) {
 		if (p != null && (p.isOp() || p.hasPermission("namelayer.admin"))) {
 			return true;
 		}
-		if (group == null || player == null || perm == null) {
```

Comment:
Why remove this null check

Reply:
If the player was null then Bukkit.getPlayer would've already thrown an error, so it's redundant

Reply:
i didn't want to change the logic

Reply:
Ok yeah, right